project_management/estimating.md

```diff
@@ -0,0 +1,65 @@
+# Estimating
+
+Estimates are key to what we do.
+
+We are in the business of selling software creation hours. We want to do this as inexpensively as possible while maintaining a healthy revenue for growth and downtimes. As a group, we need to hold appropriate tension between cost and quality.
+
+Estimates should be made by two developers at a minimum; estimates are shared, and ideally by the implementing team.
```

Comment:
`estimates are shared` meaning:
- We do them together,
- We tell the team what we estimated, or
- Everyone agrees on the estimates?

Reply:
It is badly phrased. It means that if we both estimate something and later on implement it, we are working on an estimate that we've done and therefore we will feel more responsible for it. We are succeeding or not as a team.
I'll change it as bit.